deploy/eck-operator/values.yaml

```diff
@@ -130,6 +130,8 @@ webhook:
   # This is required to allow for communication with the kube API when using some alternate CNIs in conjunction with webhook enabled.
   # CAUTION: Proceed at your own risk. This setting has security concerns such as allowing malicious users to access workloads running on the host.
   hostNetwork: false
```

Comment:
Not from this PR but would it make sense to move `hostNetwork` as a "top level" field?
`hostNetwork` is part of the `Pod` specification, as `resources`, `podSecurityContext`, `serviceAccount`... and is not directly related to the webhook configuration.

Reply:
There is also a typo in the comment:

```diff
--- a/deploy/eck-operator/values.yaml
+++ b/deploy/eck-operator/values.yaml
@@ -126,7 +126,7 @@ webhook:
   # objectSelector corresponds to the objectSelector property of the webhook.
   # Setting this restricts the webhook to act only on objects that match the selector.
   objectSelector: {}
-  # HostNetwork allows a Pod to use the Node network namespace.
+  # hostNetwork allows a Pod to use the Node network namespace.
   # This is required to allow for communication with the kube API when using some alternate CNIs in conjunction with webhook enabled.
   # CAUTION: Proceed at your own risk. This setting has security concerns such as allowing malicious users to access workloads running on the host.
   hostNetwork: false
```

Reply:
> Not from this PR but would it make sense to move hostNetwork as a "top level" field?
hostNetwork is part of the Pod specification, as resources, podSecurityContext, serviceAccount... and is not directly related to the webhook configuration.

You're absolutely right, I missed that in #6636.

Reply:
I had it at the root, but was asked to change. Will move it to the root :)

Reply:
Thanks. Also need to update:
```diff
diff --git a/deploy/eck-operator/templates/statefulset.yaml b/deploy/eck-operator/templates/statefulset.yaml
index e17c34a82..16d66e5e2 100644
--- a/deploy/eck-operator/templates/statefulset.yaml
+++ b/deploy/eck-operator/templates/statefulset.yaml
@@ -117,7 +117,7 @@ spec:
         {{- with .Values.volumes }}
           {{- toYaml . | nindent 8 }}
         {{- end }}
-      {{- if .Values.webhook.hostNetwork }}
+      {{- if .Values.hostNetwork }}
```